adrs/co2.json

```diff
@@ -0,0 +1,123 @@
+{
+    "stac_version": "1.0.0",
+    "stac_extensions": [],
+    "type": "Feature",
```

Comment:
I think there should be an STAC example both for the collection (co2) and for specific items (s3://covid-eo-data/xco2-base/xco2_16day_base.2022-01-07.tif, not sure if that's a valid date just an example, but items I think should refer to a single tif asset). I think this example might be trying to do both is that right?

Reply:
I second this, I think we need an example of a stac collection document and an item document. I'd be into a working session if we wanted to hash things out over an individual dataset.